### PR TITLE
Update 663d5bebe2eef6128a0b1e75.md

### LINKED ISSUE #54930: Update description for step 6 of Review DOM project
### Describe the Issue

A lot of campers are forgetting to use the `innerText property` when they need to reset the game in step 6 of the review DOM project. 

We are getting a lot of answers like this

```js
function resetGame() {
  playerScore = 0;
  computerScore = 0;
  playerScoreSpanElement = playerScore;
  computerScoreSpanElement = computerScore;
...
```

I think updating the description will help with this.

## updated description

````md
If the player or computer has won the game, there should be an option to reset the game and play again.

Complete the `resetGame` function that accomplishes the following:

- Resets the player and computer scores to `0`.
- Updates the `playerScoreSpanElement` and `computerScoreSpanElement` to display the new scores.
- Hides the `resetGameBtn` button.
- Shows the `optionsContainer` so the player can play again.
- Clear the content for the `winnerMsgElement` and `roundResultsMsg` elements.

**Tips**

- You can use the `innerText` property to update the content of an element. To clear the content of an element, you can set the `innerText` to an empty string.

Once you apply those changes, you will have completed the Rock, Paper, Scissors game!
````

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/review-dom-manipulation-by-building-a-rock-paper-scissors-game/step-6

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ARMummert**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54930 & #54926

I changed the description for step 6 of the Review DOM project.  I used the updated example given in the issue.  
I also changed the sentence in learn-introductory-javascripts-by-building-a-pyramid-generator from declare a character variable in your code to -> Use the `let` keyword to declare a variable called `character`.

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ let hello;
 
 Variable naming follows specific rules: names can include letters, numbers, dollar signs, and underscores, but cannot contain spaces and must not begin with a number.
 
-Declare a `character` variable in your code. 
+Use the `let` keyword to declare a variable called `character`.
```
Comment (JoyShaheb) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md:22]: please put it back to its original form, keep solution consistent to the ticket description and solution provided

**Review by JoyShaheb — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md (hunk 1 above)
Comment (ARMummert) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md:22]: Both the issues I worked on are in this PR (Issue https://github.com/freeCodeCamp/freeCodeCamp/issues/54930 & https://github.com/freeCodeCamp/freeCodeCamp/issues/54926).  This file is part of issue #54926. Per the ticket descriptions and solution provided it matches the issue.  When I tried to submit a second PR for this second issue,  it had already been committed to this PR.  Did I complete the PR incorrectly? 

Should I go ahead and revert back to the original for issue #54926?

Comment (JoyShaheb) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md:22]: okay in this case, keep it as it is then, my bad, didn't see both issues linked together.

**Review by JoyShaheb — APPROVED** [review]
good work 👍

**Comment by gikf** [comment]
Hello,

Firstly, thank you for submitting this pull request!

As you navigate through the process, we have a PR checklist to ensure consistency and quality in our contributions. We kindly ask that you genuinely follow through with each point. This not only facilitates the review process but also demonstrates a mutual respect for the community's efforts.

If you're unfamiliar with certain aspects, our [contributing guidelines](https://contribute.freecodecamp.org) are a helpful resource to get you up to speed.

<details>
<summary>**Friendly Pointers (click to expand)**</summary>

1. **Editing on GitHub:** While it's possible to edit files directly on GitHub, it's typically better not to. This helps avoid inadvertent mistakes like typos that can disrupt tests.

2. **Pull Request Title:** Please ensure the PR title follows [our convention](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title).

3. **Linking Issues:** Please ensure you link issues using the designated method. Simply update the `XXXXXX` in the PR description to include the issue number. This keeps our records organized and clear.

4. **Engaging with the Team:** We know you're eager, but kindly keep mentions and review requests limited. Our maintainers are always on the lookout and will attend to PRs in the order they come in.

5. **Branch Management:** It's a good practice not to work directly off your `main` branch. Creating separate branches for different changes allows you to smoothly update your PR even as the main repository progresses.

</details>

Please note, there's no need to close this PR. If you have questions or need guidance refining your contribution, don't hesitate to ask. Our community is here to assist.

Thank you for your enthusiasm in contributing to our project. We eagerly await more contributions from you!

**Happy Contributing!** 🌟

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md (hunk 1 above)
Comment (gikf) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ee6e3a242da6bd579de69.md:22]: > Both the issues I worked on are in this PR (Issue #54930 & #54926). This file is part of issue #54926. Per the ticket descriptions and solution provided it matches the issue. When I tried to submit a second PR for this second issue, it had already been committed to this PR. Did I complete the PR incorrectly?
> 
> Should I go ahead and revert back to the original for issue #54926?

That's caused by making both changes on the same branch - `main`. It's recommended to create new, separate branch for each separate change (issue), while keeping the `main` branch in sync with the `freeCodeCamp:main` branch. That way it's always possible to create new _clean_ branch off the `main`, so changes from different branches don't mix up.

For similar reason usually it's better to keep single PR for single issue (unless single change completes multiple issues). As there was in the meantime submitted another PR, making changes for #54930, it'd be great if you could revert change for #54930, and leave one for #54926. Making your PR close #54926, while the other PR closes #54930.

**Pull request closed without merging**